.travis.yml

```diff
@@ -9,23 +9,51 @@ matrix:
     - php: 5.6
     - php: 5.5
     - php: 5.4
-    - php: hhvm
```

Comment:
In place of the `hhvm` job, could you add a `7.2` job above the existing `7.1` job please

Reply:
Done.